src/rtk/features/userInfo/userInfoApi.ts

```diff
@@ -0,0 +1,43 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { getCookie } from '@/resources/Cookie/CookieMethods';
+
+interface UserInfoResponse {
+  party: {
+    name: string;
+  };
+}
+interface ReporteeResponse {
+  name: string;
+}
+
+export const userInfoApi = createApi({
+  reducerPath: 'userInfoApi',
+  baseQuery: fetchBaseQuery({
+    baseUrl: '/accessmanagement/api/v1/user/',
+    prepareHeaders: (headers) => {
+      headers.set('content-type', 'application/json; charset=utf-8');
+      headers.set('X-XSRF-TOKEN', getCookie('XSRF-TOKEN'));
+      return headers;
+    },
+  }),
+  endpoints: (builder) => ({
+    getUserInfo: builder.query<any, void>({
```

Comment:
Could we replace this `any` with the returned type? Looks like both of these requests return type `ReporteeResponse`.

Reply:
I agree 🙌

Reply:
Absolutely. I was leaning a litte to much on code generated by Copilot here. Specifying return types enhances clarity 👍.